service/local-dev/run_server.sh

```diff
@@ -53,8 +53,8 @@ elif [[ ${useAouUnderlays} ]]; then
   export TANAGRA_EXPORT_SHARED_GCS_BUCKET_PROJECT_ID=broad-tanagra-dev
   export TANAGRA_EXPORT_SHARED_GCS_BUCKET_NAMES=broad-tanagra-dev-bq-export
   # specify access-control
-  export TANAGRA_ACCESS_CONTROL_BASE_PATH=https://all-of-us-workbench-test.appspot.com
-  export TANAGRA_ACCESS_CONTROL_MODEL=AOU_WORKBENCH
+  export TANAGRA_ACCESS_CONTROL_BASE_PATH=https://api-dot-all-of-us-workbench-test.appspot.com
+  # export TANAGRA_ACCESS_CONTROL_MODEL=AOU_WORKBENCH
```

Comment:
nit: The base path won't be used unless you specify an access control model that uses it (`OPEN_ACCESS` is the default and does not use it). Can you either remove both of these lines, or uncomment both?

Reply:
commented both lines and added a comment to uncomment for testing :-)